image-index.md

```diff
@@ -105,13 +105,15 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 ## Platform Variants
 
 When the variant of the CPU is not listed in the table, values are implementation-defined and SHOULD be submitted to this specification for standardization.
-
-| ISA/ABI         | `architecture` | `variant`   |
-|-----------------|----------------|-------------|
-| ARM 32-bit, v6  | `arm`          | `v6`        |
-| ARM 32-bit, v7  | `arm`          | `v7`        |
-| ARM 32-bit, v8  | `arm`          | `v8`        |
-| ARM 64-bit, v8  | `arm64`        | `v8`        |
+These values SHOULD match (or be similar to) their analog listed in [the Go Language document][go-environment2].
```

Comment:
> (or be similar to)

This part makes me nuts, but I felt like I had to somehow address that `GOARM` does not use a `v` prefix, `GOARM64` will not (at least initially) support `v8`, we don't do extra things like `softfloat` or `sse2` in _this_ field, etc SOMEHOW in this wording.

Reply:
Hm, i would be more explicit in tags that are supported. One can reference go constants as valid ones, but specify them explicitly.

There will always be variants that are not yet specified or relevant for go, but do make sense to be explicit for container images that contain code that was written in other languages. For example, whilst GOARM64 is already defined, it doesn't actually do anything different in go - but it does for gcc for example.

Reply:
The table below does include explicit examples, though :sweat_smile: